Bot/constants.py

```diff
@@ -19,17 +19,20 @@
 CMD_CANCEL = 'cancel'
 
 # Logger Format
-LOGGER_FORMAT = '%(asctime)s %(clientip)-15s %(user)-8s %(message)s'
+# LOGGER_FORMAT = '%(asctime)s %(clientip)-15s %(user)-8s %(message)s'
```

Comment:
Maybe we can remove commented code. If we have a need to retrieve it, we can look at the previous versions of the code.